.github/workflows/main_workflow.yml

```diff
@@ -213,17 +213,11 @@ jobs:
           docker pull trufflesuite/ganache:beta
 
           # build required images
-          pip install .
-          autonomy fetch --local --service valory/hello_world
 
-          cd hello_world
-          autonomy build-images
-          autonomy build-images --dev
-          autonomy build-images --dependencies
-          autonomy build-images --dependencies --version 1.0.0
-          autonomy deploy build ../deployments/keys/hardhat_keys.json
-
-          cd ..
+          skaffold build --push=false
+          skaffold build -p release --push=false
+          docker build deployments/Dockerfiles/tendermint/ -t valory/open-autonomy-tendermint:0.1.0 --file deployments/Dockerfiles/tendermint/Dockerfile
+          docker build deployments/Dockerfiles/tendermint/ -t valory/open-autonomy-tendermint:1.0.0 --file deployments/Dockerfiles/tendermint/Dockerfile
```

Comment:
why again?

Reply:
IT's already in the skaffold file

Reply:
We're tagging `latest` in the skaffold, and we need some other tags. Again just need to do this until the release, then we can just tag the existing image.